certified-connectors/Polaris PSA/apiProperties.json

```diff
@@ -0,0 +1,78 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "{{ client id }}",
+          "scopes": [
+            "API"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://global.replicon.com/!/oauth2/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://global.replicon.com/!/oauth2/token"
+            },
+            "refreshUrl": {
+              "value": "https://global.replicon.com/!/oauth2/token"
+            }
+          }
+        }
+      },
+      "siteUrl": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "false"
+          },
+          "description": "Polaris Base Url (Example: na5.replicon.com)",
+          "displayName": "Polaris Base URL",
+          "tooltip": "Provide Your Polaris Base Url"
+        }
+      }
+    },
+    "iconBrandColor": "#FFFFFF",
```

Comment:
@repl-anand-vikram White background color is not allowed, please update it to some other color.

Documentation - [Design an icon for your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector)

Reply:
@repl-anand-vikram, Can you please fix this to get this PR approved